CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### 💥 Breaking Changes
 
+- Upgraded to TypeScript v4.6.4
```

Comment:
No public interfaces seem to have been changed with the TypeScript version update, I don't think it's a breaking change